yaml/pois.yaml

```diff
@@ -769,6 +773,16 @@ filters:
     output:
       <<: *output_properties
       kind: level_crossing
+  # filter the waterway / boat fuel before generic amenity=fuel, as they appear
+  # to be often tagged with both.
+  - filter:
+      any:
+        - waterway: fuel
+        - "seamark:small_craft_facility:category": fuel_station
+    min_zoom: { clamp: { min: 0, max: 16, value: { sum: [ { col: zoom }, 2.7 ] } } }
```

Comment:
should this be min zoom of say 14 instead of 0? Seems less important than general fuel which I don't think is clamped?

Reply:
OK. Updated in fd0f922.